opentelemetry-dynatrace/examples/dynatrace/Cargo.toml

```diff
@@ -9,7 +9,7 @@ futures = "0.3"
 http = "0.2"
 lazy_static = "1.4"
 opentelemetry = { version = "0.19", default-features = false, features = ["trace"] }
-opentelemetry-dynatrace = { path = "../../opentelemetry-dynatrace" }
+opentelemetry-dynatrace = { path = "../.." }
```

Comment:
While at this, could you also modify the opentelemetry dependency to be the latest (path) instead of pinned to "0.19".

Reply:
I don't think dynatrace has anyone maintaining it currently. We could consider dropping it.

Reply:
That makes sense. From their docs, it looks like OTLP is natively supported, so a custom exporter is not needed to be maintained.
https://opentelemetry.io/ecosystem/vendors/

I'll just open an issue to see if any objections.